src/screens/Disputes/Disputes.res

```diff
@@ -40,51 +66,51 @@ let make = () => {
     }
   }
   React.useEffect(() => {
-    getDisputesList()->ignore
+    if filters->isNonEmptyValue {
+      getDisputesList()->ignore
+    }
     None
-  }, [])
+  }, (filters, searchText))
```

Comment:
Why are we using` (filters, searchText)` and not [filters, searchText]

Reply:
both work similar in rescript

Reply:
If both works the same then we can replace it with [filters, searchText]